Trigger blackbox tests with label

licitdev opened the pull request:
## Scope

What's changed:

- Trigger blackbox tests with `Run Blackbox` label

## Potential Risks / Drawbacks

- 

## Review Notes / Questions

-

licitdev committed:
Trigger blackbox tests with label

danielBreitlauch reviewed (approved):
Implementation LGTM.
Not sure about the usability/use case/process implications. How people should use that.
Perhaps having it in a document explained would be good. Not sure if in public Directus or an team onboarding etc document.

licitdev committed:
Add docs

paescuj committed:
Separate PR trigger/check into own workflow file

- Nicer check reporting in pull requests
- Deduplicate if condition
- Reduce trigger types to appropriate ones

licitdev commented on docs/contributing/tests.md:
```diff
@@ -114,6 +114,11 @@ TEST_SAVE_LOGS=info pnpm --workspace-root test:blackbox
 
 The log files will be available under `tests/blackbox/server-logs-*`.
 
+### Run Blackbox Tests in Pull Requests
+
+Full blackbox tests run automatically when changes are merged into `main`. To trigger blackbox tests on a pull request,
+add the `Run Blackbox` label.
+
```
@danielBreitlauch Good point! 👍 I have added this to the documentation. Does this addition suffice?

licitdev replied:
Yes very true! Updated it to the Blackbox readme in https://github.com/directus/directus/pull/24517/commits/a36072a52e6c4caae4a9d659cfe85b0b18cabf80

paescuj committed:
Merge branch 'main' into label-trigger-blackbox

licitdev committed:
Update docs location

paescuj reviewed (approved):
Updated the PR in 3a0566cbc146f884348ac7e919815d8f8ca30932, especially to get a nicer/less confusing check report when the blackbox tests are skipped:

### Before

<img width="375" src="https://github.com/user-attachments/assets/3f657ff8-f6b0-4c1b-b46b-352558a23fcb" />

### After

<img width="375" src="https://github.com/user-attachments/assets/9f319a01-b4ff-4001-a355-4b47ae2101be" />

(aligns with other jobs that are also named `Check / ...`)
